src/components/NavigationAccount.vue

```diff
@@ -64,13 +64,13 @@
 				{{ t('mail', 'Remove account') }}
 			</ActionButton>
 		</template>
+		<AccountSettings v-if="displayAccountSettings" @close="closeAccountSettings" />
```

Comment:
I'd like to see the `account` passed as prop here instead of resolving it internally via the URL parameter. This way the modal will actually work, because there currently is no ``this.$route.params.accountId``